spaces/S000063/properties/P000154.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000063
+property: P000154
+value: true
+---
+
+Since the topology is finer than the Euclidean, {S63|P2}.
```

Comment:
```suggestion
Since the topology is finer than the Euclidean, this space is {P2}.
```
Changed because otherwise we get a circular dependency, as you can check by clicking on the `{S63|P2}` link in preview mode.